system/Database/BaseBuilder.php

```diff
@@ -1736,97 +1737,48 @@ public function getWhere($where = null, ?int $limit = null, ?int $offset = 0, bo
     }
 
     /**
-     * Sets update fields for updateBatch
-     *
-     * @param string|string[] $set
-     * @param bool            $addToDefault future use
-     * @param array|null      $ignore       ignores items in set
-     *
-     * @return $this
-     */
-    protected function updateFields($set, bool $addToDefault = false, ?array $ignore = null)
```

Comment:
(memo) This method will be `public`. It is not a BC. Okay.

Reply:
Yes, This method will be especially useful for upserts. It was meant to be public from the beginning.